avoid including make.sys, which contains hardcoding settings we don't need/want
author: Miguel Dias Costa (National University of Singapore)
--- src/Makefile.orig	2016-04-19 15:26:27.373047000 +0800
+++ src/Makefile	2016-04-19 15:26:32.414229150 +0800
@@ -1,7 +1,7 @@
 #  Should be no need to change below this line
 #
 
-include ../make.sys
+#include ../make.sys
 
 OBJS  =  constants.o io.o utility.o parameters.o hamiltonian.o overlap.o \
 	 kmesh.o disentangle.o wannierise.o plot.o transport.o
